mkdocs.yml

```diff
@@ -12,6 +12,7 @@ pages:
       - Alerts Guide: guide/alerts-guide.md
       - Events Guide: guide/events-guide.md
       # - Dashboards Guide: guide/dashboards-guide.md
+      - Saved Views: guide/saved-views.md
```

Comment:
Is Guides the right place?  Or would it make more sense to put it under Dashboards?

Reply:
This is available in different types of Apps and also on dashboards, similar to Split Screen which is in Guides. I could add a link to this from some part of Dashboards.

Reply:
Added a link under Dashboards -> Reports and Components, similar to how I linked it from Monitoring.

Reply:
I think we are struggling with finding the best place for functionality that spans multiple solutions...  I don't have an immediate suggestion that feels great.  So I don't want to block this now.

Reply:
Yes, we need to think of a better place to reference the Saved Views. For now, it is fine to have them under Guides. But we need to have them referenced for every solution.